### PR TITLE
arrigo local release 1.5.2

### PR DESCRIPTION
**Pull request by sjundi-regin**

### CODE REVIEW

File: docs/services/arrigo-serverside-functions.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ description: Change Log
 
 # Change Log
 
+# 1.1.1
```
Comment (MarkusLN) [inline docs/services/arrigo-serverside-functions.md:9]: 1.1.2

Comment (sjundi-regin) [reply docs/services/arrigo-serverside-functions.md:9]: no, it was a dependabot generated PR. Which is not tested and I dont know if we gonna merge that.

Comment (sjundi-regin) [reply docs/services/arrigo-serverside-functions.md:9]: 1.1.1 is what is build and deployed to QA and that is the version Rasmus and Joakim have tested.

File: docs/index.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ description: Releases and Change Log
 
 | Release  | Version                                      | Required EXO installation | Download |
 | -------- | -------------------------------------------- | ------------------------- | --------    
-| Latest   | [1.4.85](./arrigolocalinstaller.html#1485) | EXO2023 Edition 1 build 224 | Via installer (`--latest` flag provided)<br />or [download](https://arrigoartifacts.blob.core.windows.net/arrigo/latest/ArrigoLocalInstaller-EXO2023Edition1_224-1.4.85.exe) |
-| Stable   | [1.4.76](./arrigolocalinstaller.html#1476) | EXO2023 Edition 1 build 224 | [Download](https://arrigoartifacts.blob.core.windows.net/arrigo/stable/ArrigoLocalInstaller-EXO2023Edition1_224-1.4.76.exe) |
+| Latest   | [1.5.2](./arrigolocalinstaller.html#152) | EXO2024 Edition 1 build 20 | Via installer (`--latest` flag provided)<br />or [download](https://arrigoartifacts.blob.core.windows.net/arrigo/latest/ArrigoLocalInstaller-EXO2024Edition1_20-1.5.2.exe) |
```
Comment (MarkusLN) [inline docs/index.md:13]: Is the link supposed to be live already? Cuz it isn't right now

File: docs/arrigolocalinstaller.md [hunk 3]
```diff
@@ -5,6 +5,27 @@ description: Change Log
...
+- Arrigo config transpiler: [1.5.33](./arrigo.config.transpiler.html#1533)
+
+- Arrigo Local Services  
+  - Wamp Host: [1.1.7](./services/arrigo-wamp-host.html#117)
+  - Domain Controller: [1.1.19](./services/arrigo-domain-controller.html#1119)
+  - Volumes: [1.2.5](./services/arrigo-volumes.html#125)
+  - Chart Service: [1.1.0](./services/arrigo-services-chart.html#110)
+  - State Service: [1.1.3](./services/arrigo-services-state.html#113)
+  - DB Service: [1.1.2](./services/arrigo-services-db.html#112)
+  - Server-Side Functions: [1.1.1](./arrigo-serverside-functions.html#111)
```
Comment (MarkusLN) [inline docs/arrigolocalinstaller.md:25]: change to 1.1.2

**Review by MarkusLN — APPROVED** [review]
Fix the comments and I'm happy.

File: docs/index.md (hunk 2 above)
Comment (sjundi-regin) [reply docs/index.md:13]: it is being published right now.

Comment (MarkusLN) [reply docs/index.md:13]: Works

File: docs/services/arrigo-serverside-functions.md (hunk 1 above)
Comment (MarkusLN) [reply docs/services/arrigo-serverside-functions.md:9]: Ok

**Pull request merged**